code/WorkInProgress/AnimationLibrary.dm

```diff
@@ -240,7 +240,7 @@
 
 /mob/var/last_interact_particle = 0
 
-/proc/interact_particle(var/mob/M, var/atom/movable/target)
+/proc/interact_particle(var/mob/M, var/atom/movable/target, var/turf = FALSE)
```

Comment:
why not just do an isturf check actually?

Reply:
I was under the impression it was less performant, is it not?